app/views/layouts/to_change_everything.html.erb

```diff
@@ -127,7 +127,7 @@
               <li><a href="/tce">English</a></li>
               <li><a href="/tce/espanol">Español</a></li>
               <li><a href="http://www.crimethinc.com/blog/2016/09/21/to-change-everything-in-11-more-languages/#espanol">Español<br />&nbsp;&nbsp;&nbsp;(América Latina)</a></li>
-              <li><a href="http://www.crimethinc.com/blog/2016/09/21/to-change-everything-in-11-more-languages/#farsi">فارسی</a></li>
+              <li><a href="/tce/فارسی">فارسی</a></li>
```

Comment:
this seems to render as valid html, I think it is correct and just looks like this due to the html being ltr and the text being rtl.... not sure though